diff --git a/android_webview/browser/aw_permission_manager.cc b/android_webview/browser/aw_permission_manager.cc
index bedb16b046d45b257e28e29e410afe43f5c8817c..9c1786362f02091a4cee0db325f51bfddccf2444 100644
--- a/android_webview/browser/aw_permission_manager.cc
+++ b/android_webview/browser/aw_permission_manager.cc
@@ -339,6 +339,7 @@ int AwPermissionManager::RequestPermissions(
       case PermissionType::VR:
       case PermissionType::AR:
       case PermissionType::STORAGE_ACCESS_GRANT:
+      BRAVE_PERMISSION_TYPES
       case PermissionType::CAMERA_PAN_TILT_ZOOM:
       case PermissionType::WINDOW_PLACEMENT:
         NOTIMPLEMENTED() << "RequestPermissions is not implemented for "
@@ -546,6 +547,7 @@ void AwPermissionManager::CancelPermissionRequest(int request_id) {
       case PermissionType::VR:
       case PermissionType::AR:
       case PermissionType::STORAGE_ACCESS_GRANT:
+      BRAVE_PERMISSION_TYPES
       case PermissionType::CAMERA_PAN_TILT_ZOOM:
       case PermissionType::WINDOW_PLACEMENT:
         NOTIMPLEMENTED() << "CancelPermission not implemented for "
