diff --git a/chrome/browser/extensions/extension_install_prompt.h b/chrome/browser/extensions/extension_install_prompt.h
index 74370afb2eaab80f30230b60c0b70d6d8416966e..afa99fb31383b203cafea165bec7b71323660382 100644
--- a/chrome/browser/extensions/extension_install_prompt.h
+++ b/chrome/browser/extensions/extension_install_prompt.h
@@ -101,7 +101,7 @@ class ExtensionInstallPrompt {
   class Prompt {
    public:
     explicit Prompt(PromptType type);
-    ~Prompt();
+    virtual ~Prompt();
 
     void AddPermissionSet(const extensions::PermissionSet& permissions);
     void AddPermissionMessages(
@@ -196,6 +196,7 @@ class ExtensionInstallPrompt {
     // Called when the user clicks cancel on the dialog, presses 'x' or escape.
     void OnDialogCanceled();
 
+    BRAVE_EXTENSION_INSTALL_PROMPT_H
    private:
     bool ShouldDisplayRevokeButton() const;
 
