diff --git a/components/permissions/permission_manager.cc b/components/permissions/permission_manager.cc
index a9566441647f0e8527a6b9603f663f66e49b2abb..f6eb5de162a8167e22db8ecf4cf367ab674ec40b 100644
--- a/components/permissions/permission_manager.cc
+++ b/components/permissions/permission_manager.cc
@@ -124,6 +124,7 @@ ContentSettingsType PermissionTypeToContentSettingSafe(
       return ContentSettingsType::AR;
     case PermissionType::STORAGE_ACCESS_GRANT:
       return ContentSettingsType::STORAGE_ACCESS;
+    BRAVE_PERMISSION_MANAGER_PERMISSION_TYPE_TO_CONTENT_SETTING_SAFE
     case PermissionType::CAMERA_PAN_TILT_ZOOM:
       return ContentSettingsType::CAMERA_PAN_TILT_ZOOM;
     case PermissionType::WINDOW_PLACEMENT:
