diff --git a/content/public/browser/url_data_source.cc b/content/public/browser/url_data_source.cc
index e5b60a8179eda4791b8e250a2f75ca0b24123365..77f596c0206b41c72c7d073acd46510b07a05ac8 100644
--- a/content/public/browser/url_data_source.cc
+++ b/content/public/browser/url_data_source.cc
@@ -77,7 +77,7 @@ std::string URLDataSource::GetContentSecurityPolicyObjectSrc() {
 std::string URLDataSource::GetContentSecurityPolicyScriptSrc() {
   // Note: Do not add 'unsafe-eval' here. Instead override CSP for the
   // specific pages that need it, see context http://crbug.com/525224.
-  return "script-src chrome://resources 'self';";
+  return "script-src chrome://resources chrome://brave-resources 'self';";
 }
 
 std::string URLDataSource::GetContentSecurityPolicyStyleSrc() {
