diff --git a/third_party/blink/renderer/core/html/media/autoplay_policy.cc b/third_party/blink/renderer/core/html/media/autoplay_policy.cc
index 955e551050111da593ca52f63a6dcc1d3dd221d4..babd5fbdef6ba92ade6009a4fa3ea820fdabc101 100644
--- a/third_party/blink/renderer/core/html/media/autoplay_policy.cc
+++ b/third_party/blink/renderer/core/html/media/autoplay_policy.cc
@@ -242,6 +242,7 @@ bool AutoplayPolicy::RequestAutoplayByAttribute() {
   if (IsGestureNeededForPlayback())
     return false;
 
+  BRAVE_AUTOPLAY_POLICY_REQUEST_AUTOPLAY_BY_ATTRIBUTE
   // If it's the first playback, track that it started because of autoplay.
   MaybeSetAutoplayInitiated();
 
@@ -266,6 +267,7 @@ base::Optional<DOMExceptionCode> AutoplayPolicy::RequestPlay() {
     TryUnlockingUserGesture();
   }
 
+  BRAVE_AUTOPLAY_POLICY_REQUEST_PLAY
   MaybeSetAutoplayInitiated();
 
   return base::nullopt;
@@ -310,6 +312,7 @@ bool AutoplayPolicy::IsGestureNeededForPlayback() const {
   // autoplay muted.
   return !IsEligibleForAutoplayMuted();
 }
+BRAVE_AUTOPLAY_POLICY_IS_AUTOPLAY_ALLOWED_PER_SETTINGS
 
 String AutoplayPolicy::GetPlayErrorMessage() const {
   return IsUsingDocumentUserActivationRequiredPolicy()
